sourcegraph/about-replatform
# Add code insights page & components

Closes #86 by adding the code insights page and related components.

## To Test
- Check out the branch, nuke `node_modules`, and install packages with `yarn ci`.
- Inspect `/code-insights` on desktop and mobile, and ensure that everything matches prod.
- Check other browsers (esp Safari) to make sure code charts are displaying properly.
